src/main/java/mops/termine2/services/GruppeService.java

```diff
@@ -47,4 +47,14 @@ public Gruppe loadById(Long id) {
 		return null;
 	}
 	
+	public boolean accountInGruppe(Account account, String gruppe) {
+		List<Gruppe> nutzerGruppen = loadByBenutzer(account);
+		for (Gruppe nutzerGruppe : nutzerGruppen) {
+			if (nutzerGruppe.getName().equals(gruppe)) {
+				return true;
+			}
+		}
+		return false;
+	}
+	
```

Comment:
Kann man das nicht vereinfachen indem man in der Datenbank direkt nach Benutzernamen und Gruppe sucht?
Dann müsste man nur noch prüfen, ob es ein Ergebnis gibt.